packages/cli/src/base.ts

```diff
@@ -231,13 +232,22 @@ export abstract class BaseCommand extends Command {
   async finally(arg: Error | undefined): Promise<any> {
     try {
       if (arg) {
-        console.error('received error while cleaning up', arg)
+        if (!(arg instanceof CLIError)) {
+          console.error(
+            `
+Received an error during command execution, if you believe this is a bug you can create an issue here: 
+            
+https://github.com/celo-org/developer-tooling/issues/new?assignees=&labels=bug+report&projects=&template=BUG-FORM.yml
+
+`,
+            arg
+          )
+        }
       }
+
       const kit = await this.getKit()
       kit.connection.stop()
-    } catch (error) {
-      this.log(`Failed to close the connection: ${error}`)
-    }
+    } catch (error) {}
```

Comment:
if there is an error while stopping connect it will now be silent. I dont think this is what we want.